enos/modules/test_e2e_docker/main.tf

```diff
@@ -284,6 +343,16 @@ resource "enos_local_exec" "run_e2e_test" {
     E2E_LDAP_USER_NAME            = var.ldap_user_name
     E2E_LDAP_USER_PASSWORD        = var.ldap_user_password
     E2E_LDAP_GROUP_NAME           = var.ldap_group_name
+    E2E_GCP_PRIVATE_KEY_ID        = var.gcp_private_key_id
+    E2E_GCP_PRIVATE_KEY           = var.gcp_private_key
+    E2E_GCP_PROJECT_ID            = var.gcp_project_id
+    E2E_GCP_CLIENT_EMAIL          = var.gcp_client_email
+    E2E_GCP_REGION                = var.gcp_region
+    E2E_GCP_ZONE                  = var.gcp_zone
+    E2E_GCP_TARGET_SSH_KEY        = var.gcp_target_ssh_key
+    E2E_GCP_HOST_SET_FILTER1      = var.gcp_host_set_filter1
+    E2E_GCP_HOST_SET_FILTER2      = var.gcp_host_set_filter2
+    E2E_GCP_HOST_SET_IPS          = jsonencode(var.gcp_host_set_ips)
```

Comment:
Another thing - not the cleanest thing, but you will also need to modify `enos/modules/test_e2e_docker/test_runner.sh` with these new variables.

Reply:
It worked for me without that. When is it necessary?

Reply:
In CI, we needed to run the tests from within a docker container due to some permission restrictions on github runners (you know that whole `/etc/hosts: 127.0.0.1 localhost boundary` thing -- we can't apply that change on the runners that we use). When you run it locally, we're not using this docker container unless you invoke this test_runner.sh script)